src/zjs_util.c

```diff
@@ -180,6 +177,41 @@ bool zjs_obj_get_int32(jerry_value_t obj, const char *name, int32_t *num)
     return true;
 }
 
+void zjs_copy_jstring(jerry_value_t jstr, char *buffer, jerry_size_t *maxlen)
+{
+    jerry_size_t size = jerry_get_string_size(jstr);
```

Comment:
(For example, strcpy doesn't check for NULL - it's the caller's fault if they do that.)